src/data/generated.ts

```diff
@@ -757,9 +757,9 @@ export type Task = {
   dueDate?: Maybe<Scalars['DateTime']>,
   finalizedAt?: Maybe<Scalars['DateTime']>,
   title?: Maybe<Scalars['String']>,
-  colony: Colony,
```

Comment:
The colony and the creator must not be undefined on a task, otherwise we'll run into a lot of trouble. I guess this is connected to this:
https://github.com/JoinColony/colonyDapp/pull/1945/files#diff-e9686509e232d1e2104f9c9c12b7ffcaR162

@rdig can you try to find a solution that fixes the @FIXME and doesn't require this change? Maybe we can use stub data.

Reply:
I'm actually not sure where this change comes from (I'll look into it) but this is not related to my fixme

That fixme is actually correct, as for the first notification ever on every user, unless they also created a colony, there isn't a colony or a token set, so it doesn't make sense to wait for that data to load

Reply:
So as far as I could dig up, this is coming from the code generator and the way it decides which resolvers are optional (when building the types)

The closest thing I could find is this https://github.com/dotansimha/graphql-code-generator/issues/893#issuecomment-474924086 which suggest that in order to prevent this we need to declare all resolvers in the same file _(which wouldn't really work for us in the way the server code is set up currently)_

Let me know how you want to handle this, as I'm kinda stuck on this. Maybe try to pair and find a solution for this ?

Reply:
Oh this comes from the server. Yeah OK, we'll change it there